### ISSUE TITLE
Absolute paths to values files do not work

### ISSUE THREAD

**Issue by ariep** [roboll/helmfile#400]
The following part of a helmfile (which is contained inside `helmfile.d`)
```
environments:
  myenv:
    values:
      - "/absolute/path/to/values.yaml"
```
fails to load the values file, because apparently the absolute path is appended to the path to the `helmfile.d` directory, giving `/path/to/helmfile.d/absolute/path/to/values.yaml` which is obviously not what was intended.

**Comment by ariep** [comment]
Thanks for your response!

About the use-case: the helmfile is stored in a repo together with some files with "fixed" values for the helm charts, representing a curated set of applications and integrations between them. Besides these values that we provide, there are some values that have to be set "from the outside", by our users. Those external values are mounted into the docker container  at a fixed location, so they have a well-known absolute path. The helmfile is in another part of that container, and a relative path from there to the external values file would first go four or five levels up, then a few more down. It's possible, but not natural, and I'd say more likely to break, though the absolute path could change at some point as well of course.

Using a relative path instead is a straight-forward workaround, so this bug is no dealbreaker, but at least a better error message would be a welcome improvement.

This is part of a project to allow organisations to easily self-host cloud services (see our [project website](https://openappstack.net/)). The `helmfile` project is really useful for us, so thanks!

**Comment by mumoshu** [comment]
@ariep Not sure if you're still using helmfile, but anyway... This is finally resolved as part of #549 and #610 :)
